helmfile/charts/prometheus-alerts/files/disk-perf.yaml

```diff
@@ -0,0 +1,46 @@
+groups:
+  - name: disk-performance
+    rules:
+      - alert: DiskReadWaitTimeHigh
+        annotations:
+          description: Disk {{ $labels.device }} Wait Time on {{ $labels.instance }} is {{ $value }}, check the workload or adjust this alert
```

Comment:
```suggestion
          description: Disk {{ $labels.device }} Wait Time on {{ $labels.instance }} is {{ $value }}, check the workload
```
unless you want to make the rule threshold configurable from the overwrite configs, I would not keep the `adjust this alert`

Reply:
> configurable from the overwrite configs

Should we?